README.md

```diff
@@ -1,8 +1,12 @@
 # Pipeline CRD
 
+[![Prow unit test results](https://prow.knative.dev/badge.svg?jobs=pull-knative-build-pipeline-unit-tests)](https://prow.knative.dev/?job=pull-knative-build-pipeline-unit-tests)
+[![Prow build results](https://prow.knative.dev/badge.svg?jobs=pull-knative-build-pipeline-build-tests)](https://prow.knative.dev/?job=pull-knative-build-pipeline-build-tests)
+[![Go Report Card](https://goreportcard.com/badge/knative/serving)](https://goreportcard.com/report/knative/serving)
```

Comment:
I believe this is suppose to be `https://goreportcard.com/report/knative/build-pipeline`.

Reply:
whoa thanks for catching that XD @shashwathi

Reply:
phew, still green XD and now with way better linting results thanks `knative/serving` :innocent: